02_activities/assignments/assignment.sh

```diff
@@ -21,21 +22,23 @@ unzip rawdata.zip
 # Complete assignment here
 
 # 1. Create a directory named data
-
+mkdir data
```

Comment:
What is interesting is that if you are "cd"ed in the 02_activities/assignments folder, and run your assignment.sh file, this mkdir function call should make the data folder for you IN the assignments folder. But given the rest of your PR, it didn't. 

What that tells me is that you didn't cd into the assignments folder in the first place and why data folder was created at the root of your repo.

My assumption is you opened a terminal in VSCode, and typed something like:

```
bash 02_activities/assignments/assignment.sh
```
next time try:
```
cd 02_activities/assignments
bash assignment.sh
```